test/framework/executor_itf/executor_itf.hpp

```diff
@@ -61,16 +61,21 @@ namespace iroha {
        * Execute a command as account.
        * @param cmd The command to execute.
        * @param account_id The issuer account id.
+       * @param do_validation Initializes the same parameter of command
+       * executor.
        * @return Result of command execution.
        */
       iroha::ametsuchi::CommandResult executeCommandAsAccount(
           const shared_model::interface::Command &cmd,
-          const std::string &account_id) const;
+          const std::string &account_id,
+          bool do_validation) const;
 
       /**
        * Execute a command as account.
        * @tparam SpecificCommand The type of executed specific command.
        * @param cmd The command to execute.
+       * @param do_validation Initializes the same parameter of command
+       * executor.
        * @param account_id The issuer account id.
```

Comment:
It would be great to have the same order of parameters as in function signature.